dev/make-distribution.sh

```diff
@@ -29,7 +29,7 @@ set -e
 set -x
 
 # Figure out where the Spark framework is installed
-SPARK_HOME="$(cd "`dirname "$0"`"; pwd)"
+SPARK_HOME="$(cd "`dirname "$0"`/.."; pwd)"
```

Comment:
Just a heads up: This is breaking change for tools like spark-ec2 and [Flintrock](https://github.com/nchammas/flintrock) which automatically build Spark.

I think it's totally OK since 1) this is a developer feature, and 2) this is part of the 2.0 update. Just wanted to call that out.